.github/workflows/publish.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - name: Install and Build 🔧
         run: |
-          npm ci --audit=false --legacy-peer-deps
+          npm ci --audit=false
```

Comment:
This isn't needed after the last storybook update. Legacy deps issues resolved 🎉